deps.edn

```diff
@@ -0,0 +1,18 @@
+{:paths ["src"]
+
+ :deps
+ {org.foundationdb/fdb-java           {:mvn/version "6.3.23"}}
+
+ :aliases
+ {:dev
+  {:extra-paths ["dev"]
+   :extra-deps  {org.clojure/clojure            {:mvn/version "1.11.1"}
+                 com.lambdaisland/classpath     {:mvn/version "0.0.27"}}}
```

Comment:
why are "dev" and lambdaisland/classpath needed?

Reply:
I removed `lambdaisland/classpath`. I you want me to remove `:dev` completely I can also do that, it's just nice for people who might want to use the clojure cli.